scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -4,605 +4,553 @@ import "GvsWarpTasks.wdl" as Tasks
-                gather_type = "CONVENTIONAL",
-                output_vcf_name = "${filter_set_name}.vrecalibration.gz",
-                preemptible_tries = 3,
-                gatk_override = gatk_override
-        }
-    }
-
-    if (GetNumSamplesLoaded.num_samples <= snps_variant_recalibration_threshold) {
-        call Tasks.SNPsVariantRecalibrator as SNPsVariantRecalibratorClassic {
-            input:
-                sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
-                sites_only_variant_filtered_vcf_index = MergeVCFs.output_vcf_index,
-                model_report = snps_model,
-                recalibration_filename = filter_set_name + ".snps.recal",
-                tranches_filename = filter_set_name + ".snps.tranches",
-                recalibration_tranche_values = snp_recalibration_tranche_values,
-                recalibration_annotation_values = snp_recalibration_annotation_values,
-                excluded_sites_bed = excluded_sites_bed,
-                hapmap_resource_vcf = hapmap_resource_vcf,
-                hapmap_resource_vcf_index = hapmap_resource_vcf_index,
-                omni_resource_vcf = omni_resource_vcf,
-                omni_resource_vcf_index = omni_resource_vcf_index,
-                one_thousand_genomes_resource_vcf = one_thousand_genomes_resource_vcf,
-                one_thousand_genomes_resource_vcf_index = one_thousand_genomes_resource_vcf_index,
-                dbsnp_resource_vcf = dbsnp_resource_vcf,
-                dbsnp_resource_vcf_index = dbsnp_resource_vcf_index,
-                use_allele_specific_annotations = true,
-                disk_size = large_disk,
-                machine_mem_gb = SNP_VQSR_machine_mem_gb,
-                downsampleFactor= SNP_VQSR_downsampleFactor
-        }
+  }
+
+  call Utils.MergeVCFs as MergeVCFs {
+    input:
+      input_vcfs = ExtractFilterTask.output_vcf,
+      output_vcf_name = "${filter_set_name}.vcf.gz",
+      preemptible_tries = 3,
+  }
+
+  call Tasks.IndelsVariantRecalibrator {
+    input:
+      sites_only_variant_filtered_vcf = MergeVCFs.output_vcf,
+      sites_only_variant_filtered_vcf_index = MergeVCFs.output_vcf_index,
+      recalibration_filename = filter_set_name + ".indels.recal",
+      tranches_filename = filter_set_name + ".indels.tranches",
+      recalibration_tranche_values = ["100.0", "99.95", "99.9", "99.5", "99.0", "97.0", "96.0", "95.0", "94.0", "93.5", "93.0", "92.0", "91.0", "90.0"],
+      recalibration_annotation_values = indel_recalibration_annotation_values,
+      mills_resource_vcf = mills_resource_vcf,
+      mills_resource_vcf_index = mills_resource_vcf_index,
+      axiomPoly_resource_vcf = axiomPoly_resource_vcf,
+      axiomPoly_resource_vcf_index = axiomPoly_resource_vcf_index,
+      dbsnp_resource_vcf = dbsnp_vcf,
+      dbsnp_resource_vcf_index = dbsnp_vcf_index,
+      use_allele_specific_annotations = true,
+      disk_size = "1000",
```

Comment:
no longer configurable?  haven't we needed to tweak this in the past along with machine memory size?

Reply:
I have only had to tweak the memory size in my runs.  I needed to tweak the disk size for `SplitIntervals`, tho.